### ISSUE TITLE
Can this work to convert existing config files to Excel?

### ISSUE THREAD

**Issue by sibzz** [on1dds/Excel-to-Nagios-config#1]
So i have a pretty big deployment, around 3000 hosts and more than 10k services. is it possible to convert the existing configuration to an excel sheet?

**Comment by on1dds** [comment]
Yes! I wrote that part for myself years ago, but I didn't save the code so I rewrote it this morning for you. You can find it here: [nag2xls](https://github.com/on1dds/Nagios-to-Excel-config)
Run it from the folder where your object definitions are. It will generate an excel with a tab per config file in that directory.

I looked at the results and they look fine, but I didn't test it in Nagios itself. So, I advise you to do so first.
Feedback is welcome

**Comment by sibzz** [comment]
Wow that is so kind of you! Thank you so much. I will check this out.
